extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -682,13 +682,43 @@ export class ConsentUI {
         (this.restrictFullscreenOn_ &&
           this.document_.activeElement !== this.ui_)
       ) {
-        // TODO (@torch2424) Send response back if enter fullscreen was succesful
+        const error = 'Could not enter fullscreen';
+        dev().warn(TAG, error);
+        this.sendIframeMessage_('Error', error);
         return;
       }
+      this.sendIframeMessage_('Success', 'Entering fullscreen');
```

Comment:
nit: I checked other places, let change the message type to `error` and `success` (without capital letter) to match the style with amp-ad and amp-iframe. 
Another improvement would be to declare messageType constant at the top of the file, but that can be addressed later.